ext/DynaLoader/DynaLoader_pm.PL

```diff
@@ -300,6 +307,7 @@ sub bootstrap_inherit {
 
 sub bootstrap {
     # use local vars to enable $module.bs script to edit values
+    our (@args, $module, @dirs, $file);
     local(@args) = @_;
     local($module) = $args[0];
     local(@dirs, $file);
```

Comment:
I think it would be neater to add the `our` to the `local` statements, to avoid repeating the variable names. e.g.:
```perl
    local our @args = @_;
    local our $module  = $args[0];
    local our (@dirs, $file);
```

Reply:
Done.